db/migrate/20210127210024_create_practice_editors.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePracticeEditors < ActiveRecord::Migration[5.2]
+  def change
+    create_table :practice_editors do |t|
+      t.belongs_to :practice, foreign_key: true
+      t.belongs_to :user, foreign_key: true
+      t.datetime :last_edited_at, null: true
+      t.string :email
```

Comment:
Hmmm. I thought the reason we needed the email field was to account for users that do not exist yet? We have to have something to send to the controller. If we want, I could try using an `attr_accessor`, but I don't know how it would behave with the `nested_attributes_for`.

Reply:
Okay, I was able to get the `attr_accessor` to work. Please let me know if this works for you 😄